### PR TITLE
Add information about Mailchimp usage

### PR DESCRIPTION
**Pull request by waxlamp**
This PR explains about the presence of Mailchimp and how to update the userbase.

Depends on #146.

### CODE REVIEW

**Comment by waxlamp** [comment]
> Hi @waxlamp, can you please pull from upstream. This pull request also contains text from #146. Although there are no merge conflicts, it would make reviewing easier. Thanks.

I rebased on master and did a force-push. If you refresh your view of the PR, it should just have a single commit now.

File: docs/40_development.md [hunk 1]
```diff
@@ -158,6 +158,59 @@ created as needed.)
...
+current slate of users from the freshly computed CSV file because Mailchimp does
+not have a way to perform a PUT-like operation during import (to borrow a term
+from RESTful API design), only to add new users and update existing ones.
+
+The reason to archive the contacts instead of deleting them has to do with
+Mailchimp's semantics for those actions. Deleting a user means they cannot be
+re-added to the audience, while archiving is a reversible action that retains
+all data and history and merely removes that user from the audience for purposes
+of receiving emails. Thus, we use an archive-and-reimport procedure to emulate
+the PUT-like operation we actually need.
```
Comment (kabilar) [inline docs/40_development.md:212]: Thanks @waxlamp.  Just to confirm, if a user has previously unsubscribed from the mailing list, will this archive-and-reimport procedure retain that preference?

File: docs/40_development.md [hunk 2]
```diff
@@ -158,6 +158,59 @@ created as needed.)
...
+5. Click on the `Add contacts` dropdown and select `Import contacts`.
+6. Select the `Upload a file` option, and follow the wizard steps, uploading the
+   CSV file from step 2 when prompted. Activate the `Update any existing
+   contacts` checkbox under the `Organize your contacts` step. Do not set any
+   tags during the `Tag your contacts` step. In the `Match column labels to
+   contact information` step, visually verify that the email address, first
+   name, and last name columns look correct matched. In the `Subscribe contacts
+   to marketing` step, ensure that `Subscribed` is selected in the dropdown. In
+   the `Review and complete your import`, read over the summary and ensure it is
+   correct before clicking the `Complete Import` button to finish the process.
```
Comment (kabilar) [inline docs/40_development.md:200]:
```suggestion
6. Select the `Upload a file` option, and follow the wizard steps, uploading the
   CSV file from step 2 when prompted. Activate the `Update any existing
   contacts` checkbox under the `Organize your contacts` step. Do not set any
   tags during the `Tag your contacts` step. In the `Match column labels to
   contact information` step, visually verify that the email address, first
   name, and last name columns look correctly matched. In the `Subscribe contacts
   to marketing` step, ensure that `Subscribed` is selected in the dropdown. In
   the `Review and complete your import`, read over the summary and ensure it is
   correct before clicking the `Complete Import` button to finish the process.
```

File: docs/40_development.md [hunk 3]
```diff
@@ -158,6 +158,59 @@ created as needed.)
...
+   click on the `Manage Audience` dropdown and select `Manage contacts`.
+4. Click on the `Manage audience` dropdown and select `Archive all contacts`.
+   Then follow the confirmation prompt to carry out the archiving operation.
+5. Click on the `Add contacts` dropdown and select `Import contacts`.
+6. Select the `Upload a file` option, and follow the wizard steps, uploading the
+   CSV file from step 2 when prompted. Activate the `Update any existing
+   contacts` checkbox under the `Organize your contacts` step. Do not set any
+   tags during the `Tag your contacts` step. In the `Match column labels to
+   contact information` step, visually verify that the email address, first
+   name, and last name columns look correct matched. In the `Subscribe contacts
```
Comment (kabilar) [inline docs/40_development.md:197]:
```suggestion
   name, and last name columns look correctly matched. In the `Subscribe contacts
```

File: docs/40_development.md [hunk 4]
```diff
@@ -158,6 +158,59 @@ created as needed.)
 If you need the credentials for logging into ImprovMX, speak to Roni
 Choudhury (<roni.choudhury@kitware.com>).
 
+### Mass emails with Mailchimp
+
+The Archive maintainers are able to send email to all users through the mass
+emailing functions of a dedicated Mailchimp account. In technical parlance,
+these communications are also known as "marketing email", though as a rule the
+maintainers do not conduct any actual marketing through this channel.
+nonetheless, such communications are governed by laws and regulations such as
```
Comment (kabilar) [inline docs/40_development.md:167]:
```suggestion
Nonetheless, such communications are governed by laws and regulations such as
```

File: docs/40_development.md (hunk 1 above)
Comment (waxlamp) [reply docs/40_development.md:212]: Yes.

**Review by kabilar — APPROVED** [review]

**Pull request merged**